e2e/execution_test.go

```diff
@@ -168,6 +261,71 @@ func testExecution(t *testing.T) {
 	t.Run("list", func(t *testing.T) {
 		resp, err := client.ExecutionClient.List(context.Background(), &pb.ListExecutionRequest{})
 		require.NoError(t, err)
-		require.Len(t, resp.Executions, 2)
+		require.Len(t, resp.Executions, 4)
+	})
+
+	t.Run("many executions in parallel", func(t *testing.T) {
+		var (
+			n          = 10
+			executions = make([]hash.Hash, 0)
+			taskKey    = "task1"
+			inputs     = &types.Struct{
+				Fields: map[string]*types.Value{
+					"msg": {
+						Kind: &types.Value_StringValue{
+							StringValue: "test",
+						},
+					},
+				},
+			}
+		)
+		t.Run("create executions", func(t *testing.T) {
```

Comment:
```suggestion
		t.Run("??? executions", func(t *testing.T) {
```

More description, as this test is not only about create exeucitons

Reply:
this specific sub-test is only creating the executions.